File: Vocal.Standard.Model/Request/DeleteMessageRequest.cs
namespace Vocal.Model.Request
{
    using System;
    using System.Collections.Generic;

    public class DeleteMessageRequest : Request
    {
        public string IdTalk { get; set; }
        public List<string> IdMessages { get; set; }
        public DateTime SentTime { get; set; }
        public string IdSender { get; set; }
    }
}

File: vocal-v4/src/app/models/request/deleteMessageRequest.ts
import {Request} from './request';

export class DeleteMessageRequest extends Request {

    SentTime: Date;
    IdTalk: string;
    IdSender : string;
    IdMessages: Array<string>;

    constructor(idSender, idTalk, idMessages) {
        super();
        this.IdSender = idSender;
        this.IdTalk = idTalk;
        this.IdMessages = idMessages;
        this.SentTime =new Date();
    }
}